packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/errors/error-overlay-layout/error-overlay-layout.tsx

```diff
@@ -80,13 +80,16 @@ export function ErrorOverlayLayout({
     } as React.CSSProperties,
   }
 
+  const hasFooter = footerMessage || errorCode
```

Comment:
Let's make it boolean when assigning, no need `!!hasFooter`

```suggestion
  const hasFooter = Boolean(footerMessage || errorCode)
```